package.json

```diff
@@ -113,7 +111,9 @@
     "webpack": "^2.6.1"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "babel-runtime": "^6.23.0",
```

Comment:
The bundle size did go up a noticeable amount.  I did some investigation on what would need to be done.  The `Array.from` references are actually introduced by a babel plugin, so it's affects aren't quite easy to see.

I'll close this PR and open another one as soon as I'm capable of testing my changes.